### PR TITLE
updated testnet to granada

### PR DESCRIPTION
**Pull request by gdsoumya**
Signed-off-by: Soumya Ghosh Dastidar <gdsoumya@gmail.com>

### CODE REVIEW

File: client/src/util/index.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const connectEthAccount = async () => {
 export const connectTezAccount = async () => {
   const client = new DAppClient({ name: "TEZEX" });
   const network =
-    config.tezos.conseilServer.network === "mainnet" ? "mainnet" : "florencenet";
+    config.tezos.conseilServer.network === "mainnet" ? "mainnet" : "granadanet";
```
Comment (anonymoussprocket) [inline client/src/util/index.js:56]: this should be a more generic "testnet" so it doesn't need to be changed every time a new testnet is deployed

File: contracts/deploy-erc20-fa12.js [hunk 2]
```diff
@@ -166,32 +166,32 @@ const deployEthereumContract = async (ethConfig, argument, web3) => {
 const deploy = async (store, web3) => {
   try {
     console.log("\nContract Addresses :");
-    const tezosSwapContract = await deployTezosContract(
-      tezConfig,
-      `(Pair (Pair True "${store.keyStore.publicKeyHash}") (Pair "${config.tezos.tokenContract.address}" (Pair 15 {})))`,
-      store
-    );
-    console.log(`- Tezos Swap contract at ${tezosSwapContract}`);
+    // const tezosSwapContract = await deployTezosContract(
```
Comment (anonymoussprocket) [inline contracts/deploy-erc20-fa12.js:169]: if this isn't needed anymore, please delete it. if it's needed under specific conditions, wrap it in an `if`

File: contracts/deploy-fa12.js [hunk 3]
```diff
@@ -57,7 +57,7 @@ async function mintMinimumBalance(store, contractAddress) {
     20_000,
     store.keyStore.publicKeyHash,
     bigInt(500000)
-      .multiply(10 ** 6)
+      .multiply(10 ** 8)
```
Comment (anonymoussprocket) [inline contracts/deploy-fa12.js:60]: unnamed constant

File: contracts/deploy-tezos-swap.js [hunk 4]
```diff
@@ -113,7 +113,7 @@ const deploy = async (store, web3) => {
     console.log("\nContract Addresses :");
     const tezosSwapContract = await deployTezosContract(
       tezosConfig,
-      `(Pair (Pair False (Pair "${store.keyStore.publicKeyHash}" 0)) (Pair (Pair {} 15) (Pair {} 0)))`,
+      `(Pair (Pair True (Pair "${store.keyStore.publicKeyHash}" 0)) (Pair (Pair {} 45) (Pair {} 0)))`,
```
Comment (anonymoussprocket) [inline contracts/deploy-tezos-swap.js:116]: unnamed constants

File: contracts/deploy-xtz-fa12.js [hunk 5]
```diff
@@ -47,32 +47,32 @@ const estimateFees = async (store, web3) => {
...
-        100000,
-        10000,
-        20000,
-        trim(xtzConfig),
-        conseiljs.TezosLanguageUtil.translateMichelsonToMicheline(
-          `(Pair True (Pair "${store.keyStore.publicKeyHash}" {}))`
-        ),
-        conseiljs.TezosParameterFormat.Micheline
-      );
+    // const xtzEstimate =
```
Comment (anonymoussprocket) [inline contracts/deploy-xtz-fa12.js:50]: if this isn't needed anymore, please delete it. if it's needed under specific conditions, wrap it in an if

**Review by anonymoussprocket — CHANGES_REQUESTED** [review]

File: client/src/util/index.js (hunk 1 above)
Comment (gdsoumya) [reply client/src/util/index.js:56]: This is wallet specific, beacon doesn't understand testnet it needs the network name

Comment (anonymoussprocket) [reply client/src/util/index.js:56]: in the config file there should be a `testnet` parameter with a 'granadanet' value

**Review by anonymoussprocket — APPROVED** [review]

**Pull request merged**